Python/other/Program_to_rotation_array.py

```diff
@@ -0,0 +1,16 @@
+def array_rotation(): 
+    save = []
+    rest_arr = []
+    for i in arr:
+        if i == d or i < d:
+            save.append(i)
+        else:
+            rest_arr.append(i)
+
+    rest_arr.extend(save)
+    print('rest arry =', rest_arr)
+
+arr = list(input().split())
```

Comment:
Display a message in print form as what are you taking input from the user for better UI

Reply:
Please check my code c269bdb